### PR TITLE
Firebase-Doc

### PR DESCRIPTION
**Pull request by wise4rmgod**
I saw some grammatical errors in the Firebase Android SDK Doc and corrected them

### CODE REVIEW

**Comment by wise4rmgod** [comment]
@googlebot I signed it!

**Comment by rlazo** [comment]
Hi @wise4rmgod thanks for your contributions! I've added some comments to your PR.

**Comment by wise4rmgod** [comment]
Alright, Thanks

**Comment by rosariopfernandes** [comment]
@rlazo Somehow I don't see the comments you claim to have added. Mind taking a look? :sweat_smile:

**Comment by rlazo** [comment]
> @rlazo Somehow I don't see the comments you claim to have added. Mind taking a look? 😅

Argh!!! This happens to me way too often... Thanks for the heads up @rosariopfernandes !

File: README.md [hunk 1]
```diff
@@ -161,22 +160,22 @@ The
...
 configuration](https://developer.android.com/studio/write/annotations#keep).  A common use for
-this annotation is because of reflection. These APIs should be generally **discouraged**, because
+this annotation is because of reflection. These APIs should be generally **discouraged** because
 they can't be proguarded.
 
 ### @KeepForSdk
 
 APIs that are intended to be used by Firebase SDKs should be annotated with
 `@KeepForSdk`. The key benefit here is that the annotation is *blessed* to throw
-linter errors on Android Studio if used by the developer from a non firebase
```
[the remark this answers is not in the document]
Comment (wise4rmgod) [reply README.md:171]: alright I get it

File: README.md [hunk 2]
```diff
@@ -189,18 +188,18 @@ proguard friendly, but the dependencies of Firebase SDKs may not be.
...
 rules in a proguard.txt that are honored by the developer's app while building
 the app's proguarded apk. This file typically contains the keep rules that need
-to be honored during the app' s proguarding phase.
+to be honored during the app's proguarding phase.
 
 As a best practice, these explicit rules should be scoped to only libraries
 whose source code is outside the firebase-android-sdk codebase making annotation
-based approaches insufficient.The combination of keep rules resulting from the
-annotations, the preguard.txt and the proguard.txt collectively determine the
+based approaches insufficient. The combination of keep rules resulting from the
```
[the remark this answers is not in the document]
Comment (wise4rmgod) [reply README.md:195]: 👍

File: README.md [hunk 3]
```diff
@@ -217,7 +216,7 @@ strategy](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.Resolutio
 
 ### Commands
 
-The simplest way to publish a project and all its associated dependencies is to
```
[the remark this answers is not in the document]
Comment (wise4rmgod) [reply README.md:220]: awesome

File: README.md [hunk 4]
```diff
@@ -189,18 +188,18 @@ proguard friendly, but the dependencies of Firebase SDKs may not be.
...
-based approaches insufficient.The combination of keep rules resulting from the
-annotations, the preguard.txt and the proguard.txt collectively determine the
+based approaches insufficient. The combination of keep rules resulting from the
+annotations and the proguard.txt collectively determine the
 APIs that are preserved at **runtime**.
 
 ## Publishing
 
 Firebase is published as a collection of libraries each of which either
-represents a top level product, or contains shared functionality used by one or
```
[the remark this answers is not in the document]
Comment (wise4rmgod) [reply README.md:203]: ok

File: README.md [hunk 5]
```diff
@@ -161,22 +160,22 @@ The
...
-linter errors on Android Studio if used by the developer from a non firebase
+linter errors on Android Studio when used by the developer from a non-firebase
 package, thereby providing a valuable guard rail.
 
 
 ### @PublicApi
 
 We annotate APIs that meant to be used by developers with
 [@PublicAPI](firebase-common/src/main/java/com/google/firebase/annotations/PublicApi.java).   This
-annotation will be used by tooling to help inform the version bump (major, minor, patch) that is
```
[the remark this answers is not in the document]
Comment (wise4rmgod) [reply README.md:179]: okay

File: README.md [hunk 6]
```diff
@@ -91,7 +91,7 @@ These are tests that run on a hardware device or emulator. These tests have
 access to Instrumentation APIs, give you access to information such as the
 [Android Context](https://developer.android.com/reference/android/content/Context).
 In Firebase, instrumentation tests are used at different capacities by different
-projects. Some tests may exercise device capabilities, while stubbing any calls
+projects. Some tests may exercise device capabilities while stubbing any calls
```
[the remark this answers is not in the document]
Comment (wise4rmgod) [reply README.md:94]: yeah

File: README.md [hunk 7]
```diff
@@ -248,7 +247,7 @@ it by commenting out its reference in [subprojects.cfg](subprojects.cfg).
 
 ### Code Formatting
 
-Code in this repo is formatted with the google-java-format tool. You can enable
```
[the remark this answers is not in the document]
Comment (wise4rmgod) [reply README.md]: noted

**Pull request closed without merging**